docs/src/documentation/03-components/07-interaction/collapse/03-accessibility.mdx

```diff
@@ -0,0 +1,104 @@
+  - `aria-controls` is automatically set to reference the collapsible content.
+- Focus management is handled automatically:
+  - Both the label and button are interactive and can receive keyboard focus.
+  - Click and keyboard events are properly managed.
+- State management is handled automatically:
+  - Animation is managed by the Slide component to ensure smooth transitions.
+
+### Best Practices
+
+- Always provide descriptive `expandButtonLabel` and `collapseButtonLabel` that clearly indicate the action.
+- Use clear and concise labels that describe the content being collapsed.
+- Ensure the content inside the Collapse component is logically grouped.
+- Include only related content within a single Collapse component.
+- When using multiple Collapse components in a sequence, ensure they're organized logically.
+- When using `customLabel`, ensure it includes text that can be announced by screen readers.
+- If the collapsed content includes interactive elements, make sure they're not focusable when collapsed.
+- All label texts should always be translated to the user's language.
+
+### Keyboard Navigation
+
+- **Tab**: Moves focus to the collapse header or expand/collapse button.
+- **Enter** or **Space**: When focus is on the header or button, toggles the collapsed state.
+- Focus order: Focus moves from the header to the button to the content inside (when expanded).
+- When collapsed, focus will skip over any focusable elements inside the collapsed content.
+
+### Examples
+
+#### Basic Usage with Required Accessibility Labels
+
+```jsx
+<Collapse
+  label="Flight Information"
+  expandButtonLabel="Show flight details"
+  collapseButtonLabel="Hide flight details"
+>
+  <p>Flight number: KL1234</p>
+  <p>Departure: 10:00</p>
+  <p>Arrival: 12:30</p>
+</Collapse>
+```
+
+Screen reader announces: "Flight Information, collapsed, button" when focused on the header and "Show flight details, collapsed, button" when focused on the button.
+
+#### Expanded State
+
+```jsx
+<Collapse
+  label="Passenger Details"
+  expandButtonLabel="Show passenger details"
+  collapseButtonLabel="Hide passenger details"
+  expanded={true}
+>
+  <p>Name: John Smith</p>
+  <p>Passport: AB123456</p>
+</Collapse>
+```
+
+Screen reader announces: "Passenger Details, expanded, button" when focused on the header and "Hide passenger details, expanded, button" when focused on the button.
+
+#### With Actions
+
+```jsx
+import TextLink from "@kiwicom/orbit-components/lib/TextLink";
+
+<Collapse
+  label="Filter Options"
+  expandButtonLabel="Show filters"
+  collapseButtonLabel="Hide filters"
+  actions={
+    <Button type="white" size="small">
+      Reset
+    </Button>
+  }
+>
+  <p>Filter controls...</p>
+</Collapse>;
+```
+
+Screen reader announces: "Filter Options, collapsed, button" when focused on the header, "Reset, button" when focused on the reset button, and "Show filters, collapsed, button" when focused on the button.
```

Comment:
Maybe you could also add one example with `customLabel` being some node and not just a text (as you mention it in the Best Practices section)?